server/minify.mjs

```diff
@@ -9,73 +10,73 @@ import uglifyES from '@node-minify/uglify-es';
 import htmlMinifier from '@node-minify/html-minifier';
 
 export default function minifyRoom() {
-  let roomHTML = fs.readFileSync(path.resolve() + '/client/room.html', {encoding:'utf8'});
+    let roomHTML = fs.readFileSync(path.resolve() + '/client/room.html', { encoding: 'utf8' });
 
-  return new Promise((resolve, reject) => {
-    minify({
-      compressor: cleanCSS,
-      input: [
-        'client/css/layout.css',
+    return new Promise((resolve, reject) => {
+        minify({
+            compressor: cleanCSS,
+            input: [
+                'client/css/layout.css',
 
-        'client/css/editmode.css',
+                'client/css/editmode.css',
 
-        'client/css/overlays/players.css',
-        'client/css/overlays/states.css',
+                'client/css/overlays/players.css',
+                'client/css/overlays/states.css',
 
-        'client/css/widgets/basicwidget.css',
-        'client/css/widgets/button.css',
-        'client/css/widgets/card.css',
-        'client/css/widgets/classes.css',
-        'client/css/widgets/deck.css',
-        'client/css/widgets/holder.css',
-        'client/css/widgets/label.css',
-        'client/css/widgets/pile.css',
-        'client/css/widgets/spinner.css'
-      ],
-      output: '/tmp/out.css'
-    }).then(function(min) {
-      roomHTML = roomHTML.replace(/ \{\{CSS\}\} /, min);
-      return minify({
-        compressor: process.env.NOCOMPRESS ? noCompress : uglifyES,
-        input: [
-          'client/js/domhelpers.js',
-          'client/js/connection.js',
-          'client/js/serverstate.js',
-          'client/js/editmode.js',
-          'client/js/geometry.js',
-          'client/js/mousehandling.js',
-          'client/js/statemanaged.js',
-          'client/js/widgets/widget.js',
+                'client/css/widgets/basicwidget.css',
+                'client/css/widgets/button.css',
+                'client/css/widgets/card.css',
+                'client/css/widgets/classes.css',
+                'client/css/widgets/deck.css',
+                'client/css/widgets/holder.css',
+                'client/css/widgets/label.css',
+                'client/css/widgets/pile.css',
+                'client/css/widgets/spinner.css'
+            ],
+            output: os.tmpdir() + '/out.css'
```

Comment:
Do we really need a copy of the css & js written to files? This change of creating a tmp dir will now start creating new dirs that keep getting created and not cleaned up until the OS is rebooted.

Reply:
The parameter isn't optional unfortunately. But os.tmpdir() shouldn't create a new directory. It just returns the default temp directory so it should be overwritten every time. There should probably be a better place for it though.